frontend/packages/topology/src/components/export-app/ExportApplicationModal.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { ActionGroup, Button } from '@patternfly/react-core';
+import { useTranslation, Trans } from 'react-i18next';
+import {
+  ModalTitle,
+  ModalBody,
+  ModalComponentProps,
+  ModalFooter,
+} from '@console/internal/components/factory/modal';
+
+export type ExportApplicationModalProps = ModalComponentProps & {
+  namespace: string;
+  startTime?: string;
+  onCancelExport?: () => Promise<boolean>;
+  onRestartExport?: () => Promise<boolean>;
```

Comment:
Why are these props optional?

Reply:
as there is a scenario when export is triggered but res data is not present so her we just that export is started, to observe this double click on export app btn